src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+  const stylesObject = {};
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
+  });
+
+  return stylesObject;
```

Comment:
The function `convertToObject` is correctly splitting the string into declarations and then into properties and values. However, it's important to note that the code does not handle cases with multiple colons within the property values or multiple semicolons without any styles between them. While it's not required to handle these cases for this task, it would be a good practice to consider these edge cases for a more robust function.